build.gradle.kts

```diff
@@ -7,7 +7,7 @@ plugins {
 }
 
 group = "uk.nhs.tis.trainee"
-version = "0.0.8"
```

Comment:
We don't need a version bump, there is no change to the service.

Add `[skip ci]` to the merge message and we can skip the build too.
https://docs.github.com/en/actions/how-tos/manage-workflow-runs/skip-workflow-runs